src/interfaces/role.interface.ts

```diff
@@ -1,7 +1,7 @@
 export interface Role {
   id: number;
   deleted: boolean;
-  name: string;
+  name: string | null;
   colour: string;
```

Comment:
Think these are the wrong way round. Name can't be null right just colour?

Reply:
facepalm. my bad updated now